fixturegen/src/commonTest/kotlin/TestGenFixture.kt

```diff
@@ -0,0 +1,24 @@
+import at.asitplus.testballoon.generatingFixtureFor
+import de.infix.testBalloon.framework.core.testSuite
+import io.kotest.matchers.string.shouldContain
+import kotlin.random.Random
+
+val aGeneratingSuite by testSuite {
+    { Random.nextBytes(32) }.generatingFixtureFor {
```

Comment:
Here I'd prefer more robust examples using a reproducible series of values by seeding.

And I'd also add a simple example which just invokes a constructor.